.circleci/config.yml

```diff
@@ -213,6 +213,112 @@ jobs:
           paths:
             - packages/*.tar.gz
 
+  build-dashr:
+    working_directory: ~/dashr
+    docker:
+      - image: plotly/dashr:ci
+        environment:
+          PYVERSION: python37
+          _R_CHECK_FORCE_SUGGESTS_: FALSE
+
+    steps:
+      - checkout
+
+      - run:
+          name: ️️🏭 clone and npm build core for R
+          command: |
+            python -m venv venv
+            . venv/bin/activate
+            git clone --depth 1 https://github.com/plotly/dash.git -b ${CIRCLE_BRANCH} dash-main
+            cd dash-main && pip install -e .[dev,testing] --progress-bar off && cd ..
+            git clone --depth 1 https://github.com/plotly/dashR.git -b dev dashR
```

Comment:
Don't you want to be running the tests against whatever the branch of DashR is in PR `X`?